addons/medical_gui/XEH_postInit.sqf

```diff
@@ -71,6 +73,25 @@ GVAR(selfInteractionActions) = [];
     false
 }, [DIK_H, [false, false, false]], false, 0] call CBA_fnc_addKeybind;
 
+["ACE3 Common", QGVAR(peekMedicalInfoKey), localize LSTRING(PeekMedicalInfo),
+{
+    // Conditions: canInteract
+    if !([ACE_player, objNull, []] call EFUNC(common,canInteractWith)) exitWith {false};
+
+    // Statement
+    [ACE_player, 0] call FUNC(displayPatientInformation);
+    false
+}, {
+    GVAR(peekLastOpenedOn) = CBA_missionTime;
+    [{
+        CBA_missionTime - GVAR(peekLastOpenedOn) > GVAR(peekMedicalInfoReleaseDelay)
+    }, {
+        QGVAR(RscPatientInfo) cutFadeOut 0.3;
+        false
+    }] call CBA_fnc_waitUntilAndExecute;
```

Comment:
```suggestion
    if (CBA_missionTime - GVAR(peekLastOpenedOn) > GVAR(peekMedicalInfoReleaseDelay) then {
        [{
            CBA_missionTime - GVAR(peekLastOpenedOn) > GVAR(peekMedicalInfoReleaseDelay)
        }, {QGVAR(RscPatientInfo) cutFadeOut 0.3}] call CBA_fnc_waitUntilAndExecute;
    };
    GVAR(peekLastOpenedOn) = CBA_missionTime;
```

This should prevent creating multiple WUAE's, though it's not that big of a deal.